### PR TITLE
Board class

### PR DESCRIPTION
**Pull request by amsmyth1**
Created board class and test code
Created readable attributes: cells
Created methods: valid_placement (sans overlap), consecutive?

### CODE REVIEW

File: test/board_test.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+require 'pry'
+
+class BoardTest < MiniTest::Test
+  def test_it_exists
+    board = Board.new
+    assert_instance_of Board, board
+  end
+
+  def test_it_has_readable_attributes
+    board = Board.new
```
Comment (jbranham1) [inline test/board_test.rb:16]: How do you feel about putting a line between our board declaration and asserts?

File: test/board_test.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  def test_consecutive_vertical
+    board = Board.new
+    cruiser = Ship.new("Cruiser", 3)
+
+    assert_equal false, board.consecutive?(cruiser, ["A1", "B2", "C4"])
+    assert_equal false, board.consecutive?(cruiser, ["A1", "B1", "D1"])
+    assert_equal true, board.consecutive?(cruiser, ["A1", "B1", "C1"])
+  end
```
Comment (jbranham1) [inline test/board_test.rb:72]: Can we remove the two empty lines here?

Comment (jbranham1) [reply test/board_test.rb:72]: Lines 71 and 72

**Review by jbranham1 — CHANGES_REQUESTED** [review]
I found just a few things in the board test that could be adjusted for syntax reasons!

File: test/board_test.rb (hunk 1 above)
Comment (amsmyth1) [reply test/board_test.rb:16]: Good catch! line removed.

File: test/board_test.rb (hunk 2 above)
Comment (amsmyth1) [reply test/board_test.rb:72]: great catch, now our code is nice and neat.

**Review by jbranham1 — APPROVED** [review]
Looks so good 💪🏼

**Pull request merged**